src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpCollectionExpressionRewriter.cs

```diff
@@ -296,6 +325,11 @@ void AddCommaIfMissing()
                     var commaToken = Token(SyntaxKind.CommaToken)
                         .WithoutLeadingTrivia()
                         .WithTrailingTrivia(TriviaList(trailingWhitespaceAndComments).AddRange(triviaAfterComma));
+
+                    // Strip trailing whitespace after the last comma.
```

Comment:
Why only after the last one? Should we instead remove from any which appears as the last element on a line?

Reply:
So all the intermediary commas are fine (they don't have a trailing whitespace).  This *Very final* one can, because it captures existing trivia from the starting collection which might have it.  For example, if you have `new C { A, B, C, }`  and then we're adding new multiline constructs to the collection.

Reply:
will add docs.